js/main.js

```diff
@@ -942,7 +942,7 @@ function BlackboxLogViewer() {
             loadFiles(files);
 
             // Clear the files, in this way we can open a file with the same path/name again
-            e.target.files.clear();
+            e.target.value = "";
```

Comment:
Have you tested this in the standalone? I tested several ways until I fond the `clear()` but I don't remember if I tested the `value = ""` way.

Reply:
I've seen that you say that in some version this has changed... Is now working with your change?

Reply:
It seems to work also in standalone. Tested this way:
1. Open a .bbl-file
2. Replace the file with another file with the same name, so that the path becomes identical.
3. Open the same file name again

Reply:
Yes, then it seems to work. It is not needed to change the file, it must load again the same file if you ask for it.